test/e2e-tests.sh

```diff
@@ -117,7 +117,9 @@ go_test_e2e -timeout=2m ./test/e2e/multicontainer || failed=1
 toggle_feature multi-container Disabled
 
 toggle_feature responsive-revision-gc Enabled
+immediate_gc
 go_test_e2e -timeout=2m ./test/e2e/gc || failed=1
+default_gc
```

Comment:
Should this rather fetch the gc CM before manipulating it and then setting it back to the before state here, rather than assuming a default that's "hardcoded"?

Reply:
That's a good idea, but I dread writing shell scripts.

My hope was to do the dumb thing here and replace it once https://github.com/knative/pkg/issues/1563 comes to fruition.

Reply:
something like that?

Reply:
Yep.